src/bit_set.rs

```diff
@@ -15,26 +15,21 @@ use std::slice;
 #[derive(Default,Debug)]
 pub struct Bitset {
     head: usize,
-    tail: Option<Box<Vec<usize>>>,
+    tail: Option<Vec<usize>>,
 }
 
 fn bits_per_word() -> usize {
     usize::max_value().count_ones() as usize
 }
 
-#[inline(never)]
-fn clone_slow(arg: &Box<Vec<usize>>) -> Box<Vec<usize>> {
-    arg.clone()
-}
-
 impl Clone for Bitset {
     #[inline]
     fn clone(&self) -> Bitset {
```

Comment:
What is the reason for this `clone()` implementation? It looks like this is the same as the derived one. The match expression below could also be replaced with `tail: self.tail.clone()`. I can only guess you have some trick related to the `#[inline]` annotation.